bin/kafka-run-class.sh

```diff
@@ -22,6 +22,16 @@ fi
 
 base_dir=$(dirname $0)/..
 
+# run kafka-env.sh
+KAFKA_ENV=$base_dir/config/kafka-env.sh
+if [ -f $KAFKA_ENV ]; then
+    . $KAFKA_ENV
+fi
+
+if [ ! -f $JAVA_HOME/bin/java ]; then
```

Comment:
Why add this requirement on `JAVA_HOME`? Without the patch, the script works fine as long as `java` is on the path.